x-pack/plugins/security/public/views/management/edit_role/components/privileges/kibana/space_aware_privilege_form.tsx

```diff
@@ -92,21 +122,47 @@ export class SpaceAwarePrivilegeForm extends Component<Props, State> {
     const basePrivilege =
       assignedPrivileges.global.length > 0 ? assignedPrivileges.global[0] : NO_PRIVILEGE_VALUE;
 
-    const description = <p>Specify the minimum actions users can perform in your spaces.</p>;
+    const description = (
+      <p>
+        <FormattedMessage
+          id="xpack.security.views.management.editRoles.components.privileges.kibana.spaceAwarePrivilegeForm.minimumActionsUserCanPerformInYourSpacesDescription"
+          defaultMessage="Specify the minimum actions users can perform in your spaces."
+        />
+      </p>
+    );
 
     let helptext;
     if (basePrivilege === NO_PRIVILEGE_VALUE) {
-      helptext = 'No access to spaces';
+      helptext = intl.formatMessage({
+        id:
+          'xpack.security.views.management.editRoles.components.privileges.kibana.spaceAwarePrivilegeForm.noAccessToSpacesHelpText',
+        defaultMessage: 'No access to spaces',
+      });
     } else if (basePrivilege === 'all') {
-      helptext = 'View, edit, and share objects and apps within all spaces';
+      helptext = intl.formatMessage({
+        id:
+          'xpack.security.views.management.editRoles.components.privileges.kibana.spaceAwarePrivilegeForm.viewEditShareAppsWithinAllSpacesHelpText',
```

Comment:
> xpack.security.views.management.editRoles.components.privileges.kibana.spaceAwarePrivilegeForm.viewEditShareAppsWithinAllSpacesHelpText

Ugh, I believe it's a bad id: it's too detailed and contains a bunch of useless info (`views`?, `editRoles` implies both `views` and `management`, `components` is useless and doesn't say anything, `privilges.kibana` - seems to be useless since we have `spaceAwarePrivilegeForm`), we should have more laconic and useful ids. Why not just something like this?

Rule: `{plugin}.{area}.[{sub-area}].{element}`

Example: `{xpack.security}.{editRoles}.{spaceAwarePrivilegeForm}.{viewEditShareAppsWithinAllSpacesHelpText}`

I doubt that we'll have collision with this shorter form. @pavel06081991 @maryia-lapata thoughts?

Reply:
yeah, agree. we will use it for all new ids